mat/logger_controller_ble.py

```diff
@@ -28,7 +29,7 @@ def set_file_mode(self, state):
         self.file_mode = state
 
 
-class LoggerControllerBLE(LoggerController):
+class LoggerControllerBLE(LoggerController, ABC):
```

Comment:
It just occurred to, you don't need to use multiple inheritance here.  `LoggerControllerBLE` inherits from `LoggerController` which inherits from `ABC`. Therefor, LoggerControllerBLE already has ABC as a baseclass.

```suggestion
class LoggerControllerBLE(LoggerController):
```

Reply:
solved and applied, thanks! :)